src/Illuminate/Support/Number.php

```diff
@@ -103,14 +103,19 @@ public static function percentage(int|float $number, int $precision = 0, ?int $m
      * @param  int|float  $number
      * @param  string  $in
      * @param  string|null  $locale
+     * @param  int|null  $digits
      * @return string|false
      */
-    public static function currency(int|float $number, string $in = 'USD', ?string $locale = null)
+    public static function currency(int|float $number, string $in = 'USD', ?string $locale = null, int $digits = null)
```

Comment:
https://en.wikipedia.org/wiki/Fraction

Feel like `$fractions` would be better than `$digits`

Reply:
PHP has both fraction and digits in their naming convention (`NumberFormatter::FRACTION_DIGITS`) but I felt that simple `$digits` after the decimal was fine.

I can change it however we want though.

Reply:
Maybe follow the other functions and use `$precision`?